core/templates/dev/head/pages/question_player/QuestionPlayerDirective.ts

```diff
@@ -79,11 +79,11 @@ oppia.directive('questionPlayer', [
         '/pages/question_player/question_player_directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        '$rootScope', 'QuestionPlayerBackendApiService',
+        '$rootScope', '$scope', 'QuestionPlayerBackendApiService',
         function(
-            $rootScope, QuestionPlayerBackendApiService) {
+            $rootScope, $scope, QuestionPlayerBackendApiService) {
           var ctrl = this;
-          ctrl.questionPlayerConfig = ctrl.getQuestionPlayerConfig();
+          $scope.questionPlayerConfig = ctrl.getQuestionPlayerConfig();
```

Comment:
/cc @ankita240796 -- if this is a general issue, please could you check that similar instances are resolved?

Thanks!

Reply:
Sure @seanlip, I will check the same.

Hi @vinitamurthi, Thanks for fixing this. 

Can you please provide some details, if any, on how you tested that using `ctrl` resulted in not passing the value to conversation skin directive. This will help me figure out the issue better. Thanks!